suse_openstack_cloud

```diff
@@ -268,6 +268,25 @@ plog_files 0 $crowbar_db_dump_file
 rm -f $crowbar_db_dump_file
 
 #############################################################
+section_header "MariaDB"
+
+if [ -e /usr/bin/mysql ]; then
```

Comment:
I don't think this check should assume knowledge of where mysql is installed unless it really has to.  Another potential problem is that you are checking for the presence of the client whereas you are trying to capture debug from the server. Maybe something like `if which mysqld >&/dev/null` would work better, or `if rpm -qa | grep -q '^mysql.*server'`.  One downside with the latter is that `rpm -qa` is super slow, although in most cases this will have recently been run by `supportconfig` before it runs this plugin, so hopefully it would be still mostly cached in the kernel's page cache.

Reply:
`rpm -q mariadb` ? That's fast. Or `rpm -q --whatprovides mysql`